### PR TITLE
Split Util class regarding the responsabilities

### PR DESCRIPTION
**Pull request by Tithugues**
`PhpMyAdmin\Util` is a very huge class with a lot of responsabilities.

There is a lot of work to split it, but this may be just a small start, waiting for a more normalized code.

### CODE REVIEW

File: libraries/classes/Html/Forms/Fields/BrowseUploadFileBlock.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+declare(strict_types=1);
+
+namespace PhpMyAdmin\Html\Forms\Fields;
+
+
+use PhpMyAdmin\Html\Forms;
+use PhpMyAdmin\Util;
+
+/**
+ * Description
```
Comment (williamdes) [inline libraries/classes/Html/Forms/Fields/BrowseUploadFileBlock.php]: :)

File: libraries/classes/Html/Forms/Fields/BrowseUploadFileBlock.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Description
```
Comment (williamdes) [inline libraries/classes/Html/Forms/Fields/BrowseUploadFileBlock.php]: :)

File: libraries/classes/Html/Forms/Fields/BrowseUploadFileBlock.php [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+class BrowseUploadFileBlock implements FieldGenerator
+{
+
+    /**
+     * Prepare the form used to browse anywhere on the local server for a file to
+     * import
+     *
+     * @param string $max_upload_size maximum upload size
+     *
+     * @return String
```
Comment (williamdes) [inline libraries/classes/Html/Forms/Fields/BrowseUploadFileBlock.php]: `string`

File: libraries/classes/Html/Forms/Fields/BrowseUploadFileBlock.php [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+     */
+    public static function generate($max_upload_size): string
```
Comment (williamdes) [inline libraries/classes/Html/Forms/Fields/BrowseUploadFileBlock.php]: can you add a typehint for the parameter ?

File: libraries/classes/Html/Forms/Fields/DropDown.php [hunk 5]
```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Description
```
Comment (williamdes) [inline libraries/classes/Html/Forms/Fields/DropDown.php]: same everywhere

File: libraries/classes/Html/Forms/Fields/DropDownUploadFileBlock.php [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+{
+
+    /**
+     * Prepare the form used to select a file to import from the server upload
+     * directory
+     *
+     * @param ImportPlugin[] $import_list array of import plugins
+     * @param string         $uploaddir   upload directory
+     *
+     * @return String
```
Comment (williamdes) [inline libraries/classes/Html/Forms/Fields/DropDownUploadFileBlock.php]: `string`

**Review by williamdes — APPROVED** [review]

File: libraries/classes/Html/Generator.php [hunk 7]
```diff
@@ -0,0 +1,1294 @@
...
+    /**
+     * Displays a button to copy content to clipboard
+     *
+     * @param string $text Text to copy to clipboard
+     *
+     * @return string  the html link
+     *
+     * @access  public
+     */
+    public static function showCopyToClipboard($text): string
```
Comment (williamdes) [inline libraries/classes/Html/Generator.php]: `htmlspecialchars` only accepts `string` so adding `string` as a typehint might be a good idea

**Review by MauricioFauth — APPROVED** [review]

**Comment by Tithugues** [comment]
Please don't validate this PR for now. I found a display bug in the Settings page.

**Comment by Tithugues** [comment]
It seems not to be related to my changes. Switching to `master`, I also have the error on home page and in 2 first tabs of the Settings page.
So if you want to validate this PR... feel free.
Thank you.

**Review by MauricioFauth — APPROVED** [review]

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

**Pull request merged**